package.json

```diff
@@ -0,0 +1,623 @@
+{
+  "name": "portfolio",
+  "version": "1.0.0",
+  "description": "<a name=\"readme-top\"></a>",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "test": "jest --coverage",
+    "watch": "webpack --watch",
+    "start": "webpack serve --open",
+    "build": "webpack"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "css-loader": "^6.8.1",
+    "eslint": "^7.32.0",
+    "eslint-config-airbnb-base": "^14.2.1",
+    "eslint-plugin-import": "^2.27.5",
+    "hint": "^7.1.10",
+    "html-webpack-plugin": "^5.5.3",
+    "style-loader": "^3.3.3",
+    "stylelint": "^13.13.1",
+    "stylelint-config-standard": "^21.0.0",
+    "stylelint-csstree-validator": "^1.9.0",
+    "stylelint-scss": "^3.21.0",
+    "webpack": "^5.88.2",
+    "webpack-cli": "^5.1.4",
+    "webpack-dev-server": "^4.15.1"
+  },
+  "dependencies": {
+    "abab": "^2.0.6",
+    "abbrev": "^1.1.1",
+    "acorn": "^8.8.2",
+    "acorn-globals": "^7.0.1",
+    "acorn-jsx": "^5.3.2",
+    "acorn-jsx-walk": "^2.0.0",
+    "acorn-walk": "^8.2.0",
+    "agent-base": "^6.0.2",
+    "agentkeepalive": "^4.3.0",
+    "aggregate-error": "^3.1.0",
+    "ajv": "^8.12.0",
+    "ajv-formats": "^2.1.1",
+    "ansi-align": "^3.0.1",
+    "ansi-colors": "^4.1.3",
+    "ansi-regex": "^5.0.1",
+    "ansi-styles": "^4.3.0",
+    "anymatch": "^3.1.3",
+    "aproba": "^2.0.0",
+    "are-we-there-yet": "^2.0.0",
```

Comment:
- [ ] I tried to run `npm install` command in your repo but could not succeed. it gave me error. then I looked in the package.json file and surprisingly `"webpack": , "webpack-cli":` does not exist in your devDependencies in this package.json file.
kindly consider adding these in your devDependencies by following guidelines in this [link](https://webpack.js.org/guides/getting-started/)